src/components/SiteKeyboards/SiteKeyboardsPresentation.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Widget from 'components/Widget'
+
+function SiteKeyboardsPresentation({ title, widgets }) {
+  return (
+    <div
+      data-testid="SiteKeyboardsPresentation"
+      className="justify-center max-w-screen-lg px-4 pb-4 lg:px-0 lg:pb-10 mx-auto"
+    >
+      {widgets?.map((widget) => (
+        <div key={widget?.id} className="flex lg:my-4">
+          <Widget.Container data={widget} />
+        </div>
+      ))}
+      <div>
+        <section className="wysiwyg">
+          <h2>{`${title} keyboards let you type in your language!`}</h2>
+          <p>
+            Many languages have alphabets with letters that are not available on the standard English keyboard. To type
+            these letters, you must install a keyboard that will render characters in your Indigenous language. When the{' '}
+            {title} keyboard is installed on a computer, mobile phone or tablet, you will be able to type using your
```

Comment:
Are these variables formatted right? The {' '} looks strange

Reply:
Yes, It just ensures a space is maintained in the text before the inline variable `{title}` . There are a few different ways to achieve it of course but this is one.

Reply:
Thanks!